openwisp_notifications/handlers.py

```diff
@@ -192,34 +223,32 @@ def send_email_notification(sender, instance, created, **kwargs):
     if not (email_preference and instance.recipient.email and email_verified):
         return
 
-    try:
-        subject = instance.email_subject
-    except NotificationRenderException:
-        # Do not send email if notification is malformed.
+    recipient_id = instance.recipient.id
+    cache_key = f'email_batch_{recipient_id}'
+    cache_key_pks = f'{instance.recipient.email}_pks'
```

Comment:
Why did you use a separate key for storing PKs? Can we add another field to `cache_key`?